src/main/java/jalau/at18/war/view/DieLabel.java

```diff
@@ -0,0 +1,16 @@
+package jalau.at18.war.view;
+
+import javax.swing.JLabel;
+
+public class DieLabel extends JLabel {
+
+    public static final int DEFAULT_POSX = 150;
+    public static final int DEFAULT_POSY = 50;
+    public static final int DEFAULT_WIDTH = 75;
+    public static final int DEFAULT_HEIGHT = 20;
+
+    public DieLabel() {
+        super("-");
```

Comment:
Maybe, you could use other character as an example for the label, it could be a number, please; because at the first time I didn't see the character in the window until I read the code.

Reply:
Makes sense, I haven't decided what to show by default yet so I think this can be zero by default. I'll update it